src/cli/config.js

```diff
@@ -94,7 +98,16 @@ async function getConfigurationFile(filename, folder) {
         }
     }
 
-    return await getDefaultConfigurationFileTraversingParents(folder);
+    // Look at parent directories or, finally, the user's home directory
+    filepath = await getDefaultConfigurationFileTraversingParents(folder);
+    if (!filepath) {
+        const homedir_file = path.join(os.homedir(), ".svglintrc.js");
+        if (await fileExists(homedir_file)) {
+            filepath = homedir_file;
+        }
+    }
+
+    return filepath || false;
```

Comment:
Can you remove the `|| false`, I don't think it adds anything at this point?

```suggestion
return filepath;
```